### PR TITLE
Fix TypeScript errors on the clean build

### PR DESCRIPTION
**Pull request by sielay**

### CODE REVIEW

**Comment by sielay** [comment]
Regards #54 and #57

**Comment by sielay** [comment]
Literally hate netlify

File: src/store.tsx [hunk 1]
```diff
@@ -31,3 +31,5 @@ export const store = createStore<StoreState>(
   initialState,
   devToolsEnhancer({}),
 );
+
+export const ABC = true;
```
Comment (fabien0102) [inline src/store.tsx]: Why this line? and why the `.tsx` (I try to only put `tsx` if the file contain some jsx inside :wink: )

**Comment by sielay** [comment]
Line was about `file is not module` I will revert it. At the moment Netlify has problems with storybooks which doesn't build locally too. I'll fix it and then message you for review.

**Comment by i001962** [comment]
I tried to applying this PR to get beyond a typescript build error (which it does) but the PR doesn't seem to pull in menuItems and page content. The footer shows up but not the menu and page content for the homepage and here's a screen capture of localhost:8000/blog/

https://screencast.com/t/J703wVIMcl

@sielay any ideas what might need to be changed in layouts/index.tsx to make the missing items appear? It's all new to me.

**Pull request closed without merging**